server/src/lfortran-accessors.ts

```diff
@@ -206,6 +206,12 @@ export class LFortranCLIAccessor implements LFortranAccessor {
             lfortranPath);
           this.logger.error(LFortranCLIAccessor.LOG_CONTEXT, err);
         }
+        // executable-command
+        this.logger.info(
+          LFortranCLIAccessor.LOG_CONTEXT,
+          "Running lfortran with the following parameters: %s",
+          lfortranPath + " " + params.join(" ")
+        )
```

Comment:
This is part of https://github.com/lfortran/lfortran-lsp/pull/43, we can remove from either of the PRs

Reply:
I've added similar logging for lfortran commands as part of https://github.com/lfortran/lfortran-lsp/pull/42 (once it is merged, you will need to set the log level to `debug` in the extension's settings).

When you log messages, please reduce the construction of temporary objects as much as possible. Instead of string concatenations, please use positional arguments like you started to. If you need to create a temporary object but do not need to always log the message, please make the statement conditional on the log level. Finally, if you need to log command-line parameters, you should escape them for safety.

```ts
import shellescape from 'shell-escape';

// ...
    if (this.logger.isDebugEnabled()) {
      const escapedCommand: string = shellescape([lfortranPath].concat(params));
      this.logger.info(
        LFortranCLIAccessor.LOG_CONTEXT,
        "Running lfortran with the following parameters: %s",
        escapedCommand
      )
    }
// ...
```